recipes/yapsy/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "yapsy" %}
+{% set version = "1.11.223" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "45bb39a72461913388d7f3e92ce33816e84fff59946c45e16796eb1fb1d5f2a6" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```

Comment:
consider making `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Thank you